### ISSUE TITLE
[service] Provide a Github Action for authors on github

### ISSUE THREAD

**Issue by Minimonium** [conan-io/conan-center-index#555]
https://github.com/conan-io/conan-center-index/issues/550#issuecomment-569459164
>It would be great if you provided an official GH action or something similar to automate the process, since just open a PR sounds a bit error-prone (I mean, I'm pretty sure I'll forget it sooner or later).

Reference to a similar issue in the vcpkg: https://github.com/microsoft/vcpkg/issues/9168

Github Action docs: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/building-actions

**Comment by uilianries** [comment]
Good idea, but it will solve partially. Some projects are in Source Forge, or in an internal FTP server.

**Comment by Minimonium** [comment]
I've made a [Conan Request Action](https://github.com/marketplace/actions/conan-request) to automatically create [request] issues in the CCI on publishing releases.

Repository: https://github.com/Minimonium/conan-request-action
Usage example: https://github.com/Minimonium/conan-request-action-example
Example of an issue: https://github.com/conan-io/conan-center-index/issues/563

I've [played with more automation](https://github.com/Minimonium/conan-publish-action) regarding the PR generation, but it seems that the most reasonable approach is to create them from actions from CCI itself. 

Any feedback is welcomed. 🙂

**Comment by danimtb** [comment]
Thanks for the proposal @Minimonium, I really like the idea of automating things. In this case creating issues would be the most reasonable action to take as automating the PR bumping version might not be as straightforward.

On the other hand, I see this is still a too early optimization now, as we are still fixing things in the CI pipelines. Moreover, this may lead to creating new versions for libraries that are not really used, so we will have to somehow curate the repos to watch with this action. Maybe that would be helpful for the most popular ones such as boost, openssl, qt, opencv...

**Comment by Minimonium** [comment]
@danimtb We need a mechanism to restrict packages and their versions for which we create binaries. The most important thing for consumers is to have the recipe ready first and foremost. 
Regarding non-github packages - we actually may look into a cron action that pings a remote, lol.
